storage/doc.go

```diff
@@ -349,43 +350,29 @@ To create a client which will use gRPC, use the alternate constructor:
 	}
 	// Use client as usual.
 
-Using the gRPC API inside GCP with a bucket in the same region can allow for
-[Direct Connectivity] (enabling requests to skip some proxy steps and reducing
-response latency). A warning is emmitted if gRPC is not used within GCP to
-warn that Direct Connectivity could not be initialized. Direct Connectivity
-is not required to access the gRPC API.
+One major advantage of the gRPC API is that it can use [Direct Connectivity],
+enabling requests to skip some proxy steps and reducing responce latency.
+Requirements to use Direct Connectivity include:
```

Comment:
Nit: should Direct Connectivity be in brackets? Maybe that's too many links...

Reply:
I just linked to it the first time I mentioned it in the paragraph, I think that is sufficient...